<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceManager</class>
 <widget class="QDialog" name="ServiceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Service Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="servicemanager.qrc">
    <normaloff>:/images/player_play.png</normaloff>:/images/player_play.png</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="4" column="6">
    <widget class="QPushButton" name="push_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QFrame" name="frame3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="listServices">
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="indentation">
         <number>20</number>
        </property>
        <property name="uniformRowHeights">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>150</number>
        </attribute>
        <column>
         <property name="text">
          <string>SDIR</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>TAG</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Service Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Running</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Enabled</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="textTopLabel">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>The following services are available on this system</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushStart">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>31</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Start</string>
     </property>
     <property name="icon">
      <iconset resource="servicemanager.qrc">
       <normaloff>:/images/player_play.png</normaloff>:/images/player_play.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushRestart">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>31</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Restart</string>
     </property>
     <property name="icon">
      <iconset resource="servicemanager.qrc">
       <normaloff>:/images/reload.png</normaloff>:/images/reload.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushEnableStartup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>31</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Enable Service</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="pushDisableStartup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>31</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Disable Service</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushStop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>31</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Stop</string>
     </property>
     <property name="icon">
      <iconset resource="servicemanager.qrc">
       <normaloff>:/images/process-stop.png</normaloff>:/images/process-stop.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">qdir.h</include>
  <include location="local">qfile.h</include>
  <include location="local">qprocess.h</include>
  <include location="local">qmessagebox.h</include>
 </includes>
 <resources>
  <include location="servicemanager.qrc"/>
 </resources>
 <connections/>
</ui>
